build-tools/packages/build-tools/src/bumpVersion/versionSchemes.ts

```diff
@@ -0,0 +1,58 @@
+/*!
```

Comment:
I want this file to be in the version-tools project, but I can't do that until we can deprecate build-tools because they're used there.